external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -256,7 +256,7 @@ impl<Progress: Write> DependencyGraphBuilder<Progress> {
             )?;
         let dep_lock_files = dep_graphs
             .values()
-            .map(|graph_info| graph_info.g.write_to_lock(self.install_dir.clone()))
+            .map(|graph_info| graph_info.g.write_to_lock(self.install_dir.clone(), None))
```

Comment:
Lock file is created here and below for computing dependency digest--we don't want to merge the result with an existing lock file, hence `None`.

Reply:
Makes sense, could you add a comment in the code for this?